src/main/java/seedu/duke/AnimeStorage.java

```diff
@@ -0,0 +1,133 @@
+package seedu.duke;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class AnimeStorage {
+
+    /* Files */
+    private static final String RELATIVE_DIR = System.getProperty("user.dir");
+    private static final String FILE_SEPARATOR = File.separator;
+
+    private File dataFile;
+    private String[] pathnames;
+
+    //public static void main(String[] args) throws IOException {
+    //    AnimeStorage  animeStorage = new AnimeStorage("/data/AniListData");
+    //    AnimeData animeList = new AnimeData(animeStorage.readAnimeDatabase());
+    //    animeList.printAll();
+    //}
+
+    public AnimeStorage(String fileFolder) {
+        this.dataFile = new File(prepareFile(fileFolder));
+        //System.out.println(dataFile.getPath());
+        // Populates the array with names of files and directories
+        pathnames = dataFile.list();
+        // For each pathname in the pathnames array
+        //for (String pathname : pathnames) {
+        //    System.out.println(pathname);
+        //}
+    }
+
+    private String prepareFile(String fileFolder) {
+        return RELATIVE_DIR + fileFolder.replace("\\",FILE_SEPARATOR).replace("/",FILE_SEPARATOR);
+    }
+
+    public ArrayList<Anime> readAnimeDatabase() throws IOException {
+        ArrayList<Anime> animeDataList = new ArrayList<>();
+        for (String pathname : pathnames) {
+            FileReader fileData = new FileReader(dataFile.getPath() + FILE_SEPARATOR
+                    + pathname);
+            //System.out.println(fileData);
+            parseJson(animeDataList, fileData);
+        }
+        return animeDataList;
+    }
+
+    private void parseJson(ArrayList<Anime> animeDataList,FileReader  fileData) {
+        JSONParser parser = new JSONParser();
+        JSONArray jsonList = new JSONArray();
+        try {
+            jsonList = (JSONArray) parser.parse(fileData);
+        } catch (ParseException | IOException e) {
+            e.printStackTrace();
+        }
+
+        Iterator<JSONObject> iterator = jsonList.iterator();
```

Comment:
This part could be changed to: 
Iterator iterator = jsonList.iterator();

To remove the depreciated / unchecked warning